test/unit-test/core_json_utest.c

```diff
@@ -717,10 +738,6 @@ void test_JSON_Validate_Partial_Documents( void )
     jsonStatus = JSON_Validate( CUT_AFTER_OBJECT_START_MARKER,
                                 CUT_AFTER_OBJECT_START_MARKER_LENGTH );
     TEST_ASSERT_EQUAL( JSONPartial, jsonStatus );
-
-    jsonStatus = JSON_Validate( CUT_AFTER_KEY,
```

Comment:
Why was this removed?

Reply:
With the changes to the parser, a key without a value is considered illegal, not partial.  The case is also covered by one of the new tests.

Reply:
Also, I put the test back with the other illegal tests.